data-requests/README.md

```diff
@@ -0,0 +1,35 @@
+# Public Data Requests (PDR)
+
+[![super-linter](https://github.com/ccao-data/public/actions/workflows/lint.yaml/badge.svg)](https://github.com/ccao-data/public/actions/workflows/lint.yaml)
+[![pre-commit](https://github.com/ccao-data/public/actions/workflows/pre-commit.yaml/badge.svg)](https://github.com/ccao-data/public/actions/workflows/pre-commit.yaml)
+
+Public data requests are one-off requests primarily from outside the CCAO. The purpose of scripting and committing even simple requests is to ensure that request outputs are reproducible.
+
+## Workflow
+
+1. Create a new issue. Be sure to include as much of the requester's original language as possible.
```

Comment:
**issue:** This first step actually isn't necessary since the request mechanism _is_ to make an issue using the template.